services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitFormNew.tsx

```diff
@@ -577,17 +609,31 @@ export const ExplosivesPermitFormNew: FC<ExplosivesPermitFormProps &
         </Col>
       </Row>
       <Row className="flex-between form-button-container-row">
-        <Button onClick={() => setParentView(true)} className="full-mobile" type="ghost">
-          Back
-        </Button>
-        <Button type="primary" className="full-mobile" htmlType="submit" loading={props.submitting}>
-          {isHistoric ? "Submit" : "Finish And Generate Certificate"}
+        <Popconfirm
```

Comment:
I added a PopConfirm, and we show a Close button now where Back isn't appropriate.

Reply:
I feel like we over use the PopConfirm and I've developed a distaste for using it when it doesn't need to be.  Can we apply that only when it's a close action.  Otherwise it's an unnecessary UI annoyance for users.

Reply:
I would actually like to make it conditional on _the form being dirty_. I still want it on the "Back" because they could have entered data on the form.